applications/FluidDynamicsApplication/custom_elements/data_containers/low_mach_navier_stokes/low_mach_navier_stokes_data.h

```diff
@@ -0,0 +1,188 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+#pragma once
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+
+// Application includes
+#include "fluid_dynamics_application_variables.h"
+#include "custom_elements/data_containers/fluid_element_data.h"
+#include "utilities/element_size_calculator.h"
+
+namespace Kratos {
+
+///@addtogroup FluidDynamicsApplication
+///@{
+
+///@name Kratos classes
+///@{
+
+template< size_t TDim, size_t TNumNodes >
+class LowMachNavierStokesData : public FluidElementData<TDim,TNumNodes, true>
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+
+    using NodalScalarData = typename FluidElementData<TDim,TNumNodes, true>::NodalScalarData;
+    using NodalVectorData = typename FluidElementData<TDim,TNumNodes, true>::NodalVectorData;
+    using ShapeFunctionsType = typename FluidElementData<TDim,TNumNodes, true>::ShapeFunctionsType;
+    using MatrixRowType = typename FluidElementData<TDim,TNumNodes, true>::MatrixRowType;
+
+    static constexpr std::size_t BlockSize = TDim + 2;
+
+    ///@}
+    ///@name Public Members
+    ///@{
+
+    NodalVectorData Velocity;
+    NodalVectorData VelocityOldStep1;
+    NodalVectorData VelocityOldStep2;
+
+    NodalVectorData MeshVelocity;
+
+    NodalVectorData BodyForce;
+
+    NodalVectorData SolidFractionVelocity;
+
+    NodalScalarData HeatFlux;
+
+    NodalScalarData Pressure;
+    NodalScalarData PressureOldStep1;
+    NodalScalarData PressureOldStep2;
+
+    NodalScalarData Temperature;
+    NodalScalarData TemperatureOldStep1;
+    NodalScalarData TemperatureOldStep2;
+
+    double ThermodynamicPressure;
+
+    double ThermodynamicPressureDerivative;
+
+    double Resistance; // Darcy resistance (opposite to permeability, sigma)
+
+    double Conductivity; // Thermal conductivity (kappa)
+
+    double SpecificHeat; // Specific heat at constant pressure (c_p)
+
+    double HeatCapacityRatio; // Heat capacity ratio (gamma)
+
+    double DeltaTime; // Time increment
+
+    double DynamicTau; // Dynamic tau considered in ASGS stabilization coefficients
+
+    double bdf0;
```

Comment:
are these member vars meant to be `publi`c?

Reply:
Yes. Otherwise they wouldn't be accessible from the corresponding elements.